cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -75,7 +75,7 @@ const CustomSelect: React.FC<ISelectProps> = ({
       }}
     >
       {optionValues.map((opt) => (
-        <MenuItem value={opt.value} key={opt.value}>
+        <MenuItem value={opt.value} key={opt.value} disabled={opt.disabled}>
```

Comment:
Do we need to add a default here for opt.disabled?

Reply:
no, if it's falsy it won't disable it